internal/receiver/discoveryreceiver/README.md

```diff
@@ -3,20 +3,22 @@
 | Status                   |                  |
 |--------------------------|------------------|
 | Stability                | [in-development] |
-| Supported pipeline types | logs             |
+| Supported pipeline types | logs, metrics    |
 | Distributions            | [Splunk]         |
 
-The Discovery receiver is a receiver compatible with logs pipelines that allows you to test the functional
-status of any receiver type whose target is reported by an Observer. It provides configurable `status`
-match rules that evaluate the generated receiver's emitted metrics (if any), or component-level log statements
-via the instance's [zap.Logger](https://pkg.go.dev/go.uber.org/zap). It works similarly to the
+The Discovery receiver is a receiver compatible with both logs and metrics pipelines that discovers and monitors
+the functional status of services reported by Observer extensions. It works similarly to the
```

Comment:
```suggestion
The Discovery receiver is a receiver that discovers and monitors the functional status
of services reported by Observer extensions. It works similarly to the
```
Nit: Header states which pipelines are supported, we probably don't need to state it again here.